lib/matplotlib/testing/__init__.py

```diff
@@ -36,7 +36,7 @@ def setup():
                 "Could not set locale to English/United States. "
                 "Some date-related tests may fail.")
 
-    mpl.use('Agg', warn=False)  # use Agg backend for these tests
+    mpl.use('Agg', force=True, warn=False)  # use Agg backend for these tests
```

Comment:
The docstring for `mpl.use` says the use of `plt.switch_backend` is preferred over `force=True`; so why is the discouraged form being used in the tests?

Reply:
The docstring should be updated (really there is no (actual) difference between use and switch_backend anymore after this PR (other than whether we silently or loudly fail or emit a warning) so I'd rather not point to switch_backend, but I know @tacaswell did add some pointers to it at some point so I'll let him decide.

Reply:
Removing that note from the docstring is something on my to-do list that I did not get to.